melodies_monet/driver.py

```diff
@@ -902,6 +902,12 @@ def plotting(self):
             # get the plot type
             plot_type = grp_dict['type']
 
+            #read-in special settings for multi-boxplot
+            if grp == 'plot_grp6':
+                region_name = grp_dict['region_name'] #BEIMING
+                region_list = grp_dict['region_list'] #BEIMING
```

Comment:
Maybe `domain_type`/`domain_name` and `domain_list` for more consistency with the other stuff?

Reply:
For the user-facing part (e.g. the YAML), consistency is helpful.

Reply:
the problem is yaml already have "domain_type" "domain_name",

so I added "region_name" and "region_list" and "model_name_list" to differentiate from previous.

so should we change to "domain_region_name" "domain_list" and "domain_model_name_list"?